components/nds-callout/scss/callout.scss

```diff
@@ -0,0 +1,38 @@
+////
+/// @group components
+////
+
+@import '~@nice-digital/nds-core/scss/core';
+
+.callout {
+  background: $nds-colour-nice-white;
+  height: auto;
+  margin-bottom: rem($nds-spacing-large);
```

Comment:
Shouldn't this have `position: relative` so the anchor `:before` fills the space?

Reply:
So without relative it just expands to the grid so I didn't notice this!